Dockerfile

```diff
@@ -20,6 +20,9 @@ ARG ENGINE_BRANCH="master"
 # Distribution
 ARG DISTRIBUTION_BRANCH="release/2.7"
 
+# Compose CLI
+ARG COMPOSE_CLI_BRANCH="main"
```

Comment:
We'll probably have to replace this with a specific tag/reference at some point, to prevent docs going out before the binary itself is updated, but should be ok for now